docs/how-to-guides/how-to-run-chainhook-as-a-service-using-stacks.md

```diff
@@ -46,7 +46,7 @@ events_keys = ["*"]
 
 In this section, you will configure a chainhook to communicate with the network. Run the following command in your terminal and generate the `Chainhook.toml` file.
 
-`$ chainhook config generate --testnet`
+`chainhook config generate --testnet`
```

Comment:
```suggestion
```
chainhook config generate --testnet
```
```
@LakshmiLavanyaKasturi I'm curious on your thoughts for also turning these commands in to code blocks with ```. That adds a "Copy" button to the code block so developers can easily copy each command.